### ISSUE TITLE
Configurable app permissions

### ISSUE THREAD

**Issue by wojtekmaj** [pwa-builder/pwabuilder-ios#18]
Ask the developer about permissions that a given PWA will require. Otherwise, the developer needs to manually add them to Info.plist; without this action, the functions like camera or geolocation will simply fail to launch.

I tested two permissions by adding them manually to Info.plist: NSCameraUsage and NSLocationWhenInUseUsage. They work great, at least on iOS 15.

**Comment by khmyznikov** [comment]
You are right. It's not implemented here yet, but already implemented in my repo https://github.com/khmyznikov/ios-pwa-wrap/blob/main/pwa-shell/Info.plist
```
<key>NSMicrophoneUsageDescription</key>
<string>Capture Audio by user request</string>
<key>NSLocationWhenInUseUsageDescription</key>
<string>Track current location by user request</string>
<key>NSCameraUsageDescription</key>
<string>Capture Video by user request</string>
<key>BGTaskSchedulerPermittedIdentifiers</key>
```

**Comment by JudahGabriel** [comment]
Thanks for the PR, @khmyznikov! That'll add the microphone, location, and camera capabilities.

Given different apps need different capabilities, I wonder whether we should expose the capabilities in the PWABuilder iOS options dialog. Let the user check them off as needed.
